packages/react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -546,7 +543,11 @@ class ContextualMenuInternal extends React.Component<IContextualMenuInternalProp
 
   private _renderFocusZone(children: JSX.Element | null): JSX.Element {
     const { focusZoneAs: ChildrenRenderer = FocusZone } = this.props;
-    return <ChildrenRenderer {...this._adjustedFocusZoneProps}>{children}</ChildrenRenderer>;
+    return (
+      <ChildrenRenderer {...this._adjustedFocusZoneProps} role={'presentation'}>
```

Comment:
Same question, wouldn't it be ok to not specify a role here?

Reply:
Yeah for this one, I agree that it doesn’t need `role = {‘presentation’}` since it’s excluded from the accessibility tree already